tests/test_client.py

```diff
@@ -20,3 +22,30 @@
 def test_user_agent(user_agent, expected):
     client = AsyncClient(api_key='123', api_url='http:\\test', user_agent=user_agent)
     assert client.user_agent == expected
+
+
+@pytest.mark.asyncio
+@mock.patch("zyte_api.aio.client._post_func")
+async def test_request_raw_error(mock_post):
```

Comment:
It seems we can check the complete str representation of the error in this test, maybe in addition to the existing checks. Do you see any downsides to it?

Reply:
You're right. 👍 

What do you think about not including the `request_id` in the `RequestError.message` and then we update https://github.com/scrapy-plugins/scrapy-zyte-api/blob/main/scrapy_zyte_api/handler.py#L223-L224 to ensure `request_id` is logged via `er.request_id`?

Reply:
This sounds fine to me.

Reply:
We'd need to ensure to bump min python-zyte-api version in this case.